packages/itmat-ui-react/src/components/datasetDetail/tabContent/dashboard/jobs.tsx

```diff
@@ -27,6 +27,7 @@ const STATUSES: { [status: string]: any } = {
 
 const JOBTYPES: { [type: string]: any } = {
     DATA_UPLOAD: <span>Data upload</span>,
```

Comment:
@sou-chon Would it be worth renaming this at the same time to DATA_UPLOAD_CSV ?

Reply:
@fguitton i feel we shouldn't for now because there might be other places that rely on the naming....you don't want unexpected bugs when you are so close to submission..